### ISSUE TITLE
Feat: Change Vault (Workspace)

### ISSUE THREAD

**Issue by jbrea** [oflisback/obsidian-bridge.nvim#32]
In obsidian.nvim there is the `ObsidianWorkspace` command to switch to another obsidian vault (sometimes I have also two vaults open simultaneously). For obsidian-bridge.nvim to continue functioning in the new workspace, I guess this would require changing the port number and the API key of obsidian-bridge.nvim. Is there an easy way to implement this feature?

**Comment by oflisback** [comment]
I can understand it would be neat to support something like that, would you imagine it would mirror how it's done in `obsidian.nvim` with something like:

```
  instances = {
    {
      "name": "",
      "address": ""
    },
    {
      "name": "",
      "address": ""
    }
  }
```

And a command to set the active `instance`?

**Comment by jbrea** [comment]
Yes, this sounds like a great idea! Maybe `workspaces` instead of `instances` to be fully coherent with `obsidian.nvim`, and include the API key? i.e.
```
  workspaces = {
    {
      "name": "",
      "address": ""
      "rest_api_key": ""
    },
    {
      "name": "",
      "address": ""
      "rest_api_key": ""
    }
  }
```
